course7/homeworks/patrickkostjens/ls/ls.cc

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+#include <boost/filesystem.hpp>
```

Comment:
It's a good practice to order header from the most local to the most generic.
So:
- this file's header
- this project's header
- your libs
- 3rd party libs
- boost
- std